source/triggers/aws-eventbridge.txt

```diff
@@ -18,8 +25,9 @@ Overview
 MongoDB offers an `AWS Eventbridge
 <https://aws.amazon.com/eventbridge/>`_ partner event source that lets
 you send Atlas Trigger events to an event bus instead of
-calling an Atlas Function. You can configure any Trigger to send events to
-EventBridge.
+calling an Atlas Function. You can configure any Trigger type to send events to
+EventBridge. Database Triggers also support custom error handling,
+to reduce trigger suspensions due to non-critical errors. 
 
 All you need to send Trigger events to EventBridge is an AWS account ID.
 This guide walks through finding your account ID, configuring the
```

Comment:
do we want to mention the creating a custom error handler at the end of this list of things that this guide walks you through?